release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -409,6 +409,8 @@ For more information about OVN-Kubernetes as a secondary network, see xref:../ne
 [id="ocp-4-13-storage"]
 === Storage
 
+* With this update, the default credentials request for AWS has been modified to allow customer-managed keys to be used for re-encryption in the Key Management Service (KMS). For clusters with the Cloud Credential Operator (CCO) configured to use manual mode, administrators must apply those changes manually by adding `kms:ReEncrypt*` permission to their key policy. Other administrators are not impacted by this change. (link:https://issues.redhat.com/browse/OCPBUGS-5410[*OCPBUGS-5410*])
```

Comment:
If this is going to go in as an enhancement it needs an id and a heading. I would also drop the bug link. @dfitzmau

Reply:
Thanks. Updated.